common-app-lib/src/cmr/common_app/app_events.clj

```diff
@@ -54,7 +54,7 @@
   "Add a shutdown hook to call the CMR app stop command found in most if not all
    CMR microservices. Caller must supply a function that calls the application's
    stop function such as: #(cmr.search.system/stop system)"
-  [app-name stop-thread]
+  [_app-name stop-thread]
```

Comment:
as in the other pull request, I think we keep app-name because search uses them, the interface is the same across all applications, and we are adding this code soon to other apps, if I remember things correctly